backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -164,6 +165,81 @@ async def upload_boundaries_excel_sheet(
         if input_temp_file and os.path.exists(input_temp_file.name):
             os.unlink(input_temp_file.name)
 
+@router.post('/facilities',
+             summary='Upload and process facility Excel file',
+             response_description='Returns processed Excel file with validations results')
+async def upload_facilities_excel_sheet(
+        facility_file: UploadFile = File(description="Excel file containing facility data"),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)